docs/guide/user_management/customer_center/customer_center_installation_and_configuration.md

```diff
@@ -1,27 +1,13 @@
-# Customer Center - installation and configuration 
+# Customer center installation
```

Comment:
Note: this page will probably be changed completely once we know how the Customer center will be integrated into Commerce